sdk/healthinsights/health-insights-clinicalmatching-rest/README.md

```diff
@@ -0,0 +1,217 @@
+if (isUnexpected(initialResponse)) {
+throw initialResponse;
+}
+const poller = await getLongRunningPoller(client, initialResponse);
+const trialMatcherResult = await poller.pollUntilDone();
+if (isUnexpected(trialMatcherResult)) {
+throw trialMatcherResult;
+}
+const resultBody = trialMatcherResult.body;
+if (resultBody.status === "succeeded") {
+  const results = resultBody.results;
+  const patients = results.patients;
+  for (const patientResult of patients) {
+      console.log(`Inferences of Patient ${patientResult.id}`);
+      for (const tmInferences of patientResult.inferences) {
+          console.log(`Trial Id ${tmInferences.id}`);
+          console.log(`Type: ${String(tmInferences.type)}  Value: ${tmInferences.value}`);
+          console.log(`Description ${tmInferences.description}`);
+      }
+  }
+}
+else {
+  const errors = trialMatcherResult.errors;
+  if (errors) {
+      for (const error of errors) {
+          console.log('${error.code} ":" ${error.message}');
+      }
+  }
+}
+```
+## Troubleshooting
+
+### Logging
+
+Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`:
+
+```javascript
+const { setLogLevel } = require("@azure/logger");
+
+setLogLevel("info");
+```
+
+For more detailed instructions on how to enable logs, you can look at the [@azure/logger package docs](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/core/logger).
+
+## Next steps
+<!--
+This code sample show common scenario operation with the Azure Health Insights Clinical Matching library. More samples can be found under the [samples](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/healthinsights/azure-healthinsights-clinicalmatching/samples/) directory.
+
+- Match Trials FHIR: [sample_match_trials_fhir.py](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/healthinsights/azure-healthinsights-clinicalmatching/samples/sample_match_trials_fhir.py)
+
+- Match Trials Structured Coded Elements: [sample_match_trials_structured_coded_elements.py](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/healthinsights/azure-healthinsights-clinicalmatching/samples/sample_match_trials_structured_coded_elements.py)
+
+- Match Trials Structured Coded Elements Sync: [sample_match_trials_structured_coded_elements_sync.py](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/healthinsights/azure-healthinsights-clinicalmatching/samples/sample_match_trials_structured_coded_elements_sync.py)
+
+- Match Trials Unstructured Clinical Note: [sample_match_trials_unstructured_clinical_note.py](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/healthinsights/azure-healthinsights-clinicalmatching/samples/sample_match_trials_unstructured_clinical_note.py)
+-->
```

Comment:
Because the links still broken. Will change it after the merge is done and our sdk package will be in the main branch. Is there another way of doing it?

Reply:
For sample you can use relative path, I think.

Reply:
fixed